keyboards/atreus/keymaps/ibnuda/rules.mk

```
@@ -0,0 +1,9 @@
+# Build Options
+#   change yes to no to disable
+#
+
+MOUSEKEY_ENABLE  = yes     # Mouse keys(+4700)
```

We're trying to keep these firmware size notes out of the repo moving forward, as they're rarely accurate.

```suggestion
MOUSEKEY_ENABLE  = yes      # Mouse keys
```